### ISSUE TITLE
Questions about scripts "commands.sh"

### ISSUE THREAD

**Issue by Clara-mm** [ml-jku/lsc#4]
Hi Andreas, 

I am confused in the process of  implementing your code.  Could you help me solve the following problems?

What  is the difference between preprocessed data in URL (http://www.bioinf.jku.at/research/lsc/mydata.html) and compressed data in URL(http://www.bioinf.jku.at/research/lsc/compoundsTarGz/)?

What is the goal of reproducing the features and clustering?

I want to predict targets of some small molecules.  And I want to know which part of  the code in  "commands.sh" can achieve target prediction?  I am wondering that it is unnecessary for me to implement Line1-Line97 in scripts.

Thanks!

**Comment by AndreasMayr** [comment]
> What the difference between preprocessed data in URL (http://www.bioinf.jku.at/research/lsc/mydata.html) and compressed data in URL(http://www.bioinf.jku.at/research/lsc/compoundsTarGz/)?

The preprocessed data is in a binary format (that is useful for further processing steps), while the compressed data is more original raw data (csv, fpf files). The preprocessed binary data is generated from the raw data (= the csv, fpf files).

> What is the goal of reproducing the features and clustering?

Main goal is that other researchers can reproduce the results published here: https://pubs.rsc.org/en/content/articlelanding/2018/sc/c8sc00148k#!divAbstract ; Besides that, fragments of the codes published in this repository might be useful for training own target prediction models.

> I want to predict targets of some small molecules. And I want to know which part of the code in "commands.sh" can achieve target prediction? I am wondering that it is unnecessary for me to implement Line1-Line97 in scripts.

We didn't train a final model for target prediction here and do not provide any code for this here (although fragments of the codes in this repository might be useful to train an own model).  Note, the publication is mainly about method performance comparisons. Hence, we trained 3 different models in a (nested) cluster cross validation setting.
Therefore, consider, that **none** of the 3 models uses **all** the data from ChEMBL (because of the cluster cross-validation setting) for training. However, for practical use, I would recommend to train a model, that uses all data from ChEMBL (except maybe a small test set, that is kept for checking, whether something has gone wrong in the training process).
